src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MaxLatencySpec.groovy

```diff
@@ -201,34 +205,33 @@ class MaxLatencySpec extends HealthCheckSpecification {
 
         then: "Flow is rerouted and goes to the DEGRADED state"
         wait(rerouteDelay + WAIT_OFFSET) {
-            def flowHistory = flowHelper.getLatestHistoryEntry(flow.flowId)
-            flowHistory.payload.last().action == REROUTE_SUCCESS
+            def flowLastHistoryEntry = flow.retrieveFlowHistory()
+                    .getEntriesByType(FlowActionType.REROUTE).last()
+            flowLastHistoryEntry.toString() == FlowActionType.REROUTE.payloadLastAction
```

Comment:
incorrect check, should be:
assert flowLastHistoryEntry.payload.last().action == FlowActionType.REROUTE.payloadLastAction
(the same: Throttling line172-173)

Reply:
Done, fixed in MaxLatency:
```
assert flowLastHistoryEntry.payload.last().action == "Flow reroute completed"
```

+ Fixed at Throttling line172-173, but I cannot verify this test properly since it is failing at line 161 or hanging for 30 minutes